docs/definition.rst

```diff
@@ -4,14 +4,41 @@ Execution Environment Definition
 The execution environment (EE) definition file supports multiple versions.
 
   * Version 1: Supported by all ``ansible-builder`` versions.
-  * Version 2: Supported by ``ansible-builder`` versions ``1.2`` and higher.
+  * Version 2: Supported by ``ansible-builder`` versions ``2.0`` and higher.
 
 :ref:`Version 2 <version-2>` adds the capability to optionally use and verify
 signed container images. This feature is only supported with the ``podman``
 container runtime.
 
 If the EE file does not specify a version, version 1 will be assumed.
 
+.. _build-phases:
+
+Build Phases
```

Comment:
We should probably also document the actual stage names we generate, in case someone wants to do a `COPY` or something from a different stage

Reply:
👍🏻